aggregate_spatial.json

```
@@ -0,0 +1,130 @@
+{
+    "id": "aggregate_spatial",
+    "summary": "Compute zonal statistics for a vector data cube",
+    "description": "Aggregates zonal statistics for features within a vector data cube (e.g., one or multiple points or polygons) over the spatial dimensions.\n\nIn case of polygon features, the process considers all pixels for which the point at the pixel center intersects with the corresponding polygon (as defined in the Simple Features standard by the OGC).\n\nThe data cube must have been reduced to only contain two raster dimensions and a third dimension the values are aggregated for, for example the temporal dimension to get a time series. Otherwise this process fails with the `TooManyDimensions` error.\n\nThe number of total and valid pixels is returned together with the calculated values.",
+    "categories": [
+        "cubes",
+        "aggregate & resample"
+    ],
+    "experimental": true,
+    "parameter_order": [
+        "data",
+        "polygons",
+        "reducer",
+        "name",
+        "binary"
+    ],
+    "parameters": {
+        "data": {
+            "description": "A raster data cube.",
+            "schema": {
+                "type": "object",
+                "subtype": "raster-cube"
+            },
+            "required": true
+        },
+        "vector": {
+            "description": "A vector data cube containing features on which the aggregation will be based.",
+            "schema": [
+                {
+                    "type": "object",
+                    "subtype": "geojson"
+                },
+                {
+                    "type": "object",
+                    "subtype": "vector-cube"
+                }
+            ],
+            "required": true
+        },
+        "reducer": {
+            "description": "A reducer to be applied on all values of each geometry. The reducer must be a callable process (or a set of processes as process graph) such as ``mean()`` that accepts by default array as input. The process can also work on two values by setting the parameter `binary` to `true`.",
```

```suggestion
            "description": "A reducer to be applied on all values of each geometry. The reducer may be a single process such as ``mean()`` or consist of multiple sub-processes. By default, a reducer must accept an array as input. The process can also work on two values by setting the parameter `binary` to `true`.",
```
We are not using process graph when communicating with users. Also, this wording has been used throughout the processes in several places and if we want to improve the wording, we should change it everywhere, not just here.